.github/workflows/terragrunt-apply-staging.yml

```diff
@@ -23,11 +23,13 @@ env:
   TF_VAR_recaptcha_secret: ${{secrets.STAGING_RECAPTCHA_SITE_SECRET}}
   TF_VAR_recaptcha_public: 6LfJDN4eAAAAAGvdRF7ZnQ7ciqdo1RQnQDFmh0VY
   TF_VAR_notify_api_key: ${{ secrets.STAGING_NOTIFY_API_KEY }}
+  TF_VAR_cognito_notify_api_key: ${{ secrets.STAGING_NOTIFY_API_KEY }}
```

Comment:
You mentioned code duplication in the PR description. Is that part of it? Why did you not use the existing `TF_VAR_notify_api_key` ?

Reply:
I cannot use it as I have to duplicate a secret manager entry unfortunately. This is part of it